data-plane/dispatcher/src/test/java/dev/knative/eventing/kafka/broker/dispatcher/consumer/impl/OrderedConsumerVerticleTest.java

```diff
@@ -81,8 +81,8 @@ public void consumeOneByOne(final long delay, final int tasks, final int partiti
     // Mock the record dispatcher to count down the latch and save the received records order
     CountDownLatch latch = new CountDownLatch(tasks);
     final Map<TopicPartition, List<Long>> receivedRecords = new HashMap<>(partitions);
-    final var recordDispatcher = mock(RecordDispatcher.class);
-    when(recordDispatcher.apply(any())).then(invocation -> {
+    final var recordDispatcher = mock(RecordDispatcherImpl.class);
+    when(recordDispatcher.dispatch(any())).then(invocation -> {
```

Comment:
here in contrast we used to invoke the `apply`, instead of `handle`

Reply:
because ordered consumption requires to know when the dispatch ends, when unordered consumption does not